util/dates.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"regexp"
+	"time"
+)
+
+// GetDateFormmated - Returns the time in  a YYYY-MM-DD format
+func GetDateFormmated(date time.Time) string {
```

Comment:
This would be in your Service layer, as this manipulates, or helps to manipulate your data

Reply:
Updated!